sdv/constraints/tabular.py

```diff
@@ -294,13 +308,32 @@ def reverse_transform(self, table_data):
                 Transformed data.
         """
         table_data = table_data.copy()
-        diff = (np.exp(table_data[self._high]).round() - 1).clip(0)
-        low_column = table_data[self._low]
+        diff = (np.exp(table_data[self._diff_column]).round() - 1).clip(0)
 
-        if pd.api.types.is_datetime64_ns_dtype(low_column):
-            diff = pd.to_timedelta(diff)
+        if self._drop == 'high':
```

Comment:
There seems to be too much duplicated code in this `if/else` block. I would attempt to move common parts to another method